altair_pandas/_misc.py

```diff
@@ -0,0 +1,73 @@
+import altair as alt
+from typing import Iterable, Union, List
+
+toltipList = List[alt.Tooltip]
+
+
+def scatter_matrix(
+    df,
+    color: Union[str, Iterable, None] = None,
+    alpha: float = 1.0,
+    tooltip: Union[List[str], toltipList, None] = None,
+):
+    """ plots a scatter matrix
+
+    At the moment does not support neither histogram nor kde;
+    Uses f-f scatterplots instead. Interactive and with a cusotmizable
+    tooltip
+
+    Parameters
+    ----------
+    df : DataFame
+        DataFame to be used for scatterplot. Only numeric columns will be included.
+    color : string or iterable [optional]
+        Can be a column name, specific color value (hex, webcolors), or an array
+        of values to be used.
```

Comment:
I was wondering, if we need to support `array of values`. Use can always pass precreated dataframe like `scatter_matrix(df.assign(colorcol=array_values), ... )` -- having to handle column name or color value would be much simpler/cleaner?